core/client/fs/src/main/java/alluxio/client/file/cache/PageId.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * A class identifies a single cached page.
+ */
+@ThreadSafe
+public class PageId {
+  private final String mFileId;
```

Comment:
what is a file id, and why is it a string?

Reply:
fileID will be used to key the pages, any unique identifier will be ok